addons/web/static/tests/helpers/test_utils.js

```diff
@@ -490,7 +491,10 @@ function removeSrcAttribute($el, widget) {
     });
 }
 
-return session.is_bound.then(function () {
+return $.when(
```

Comment:
add a comment to explain that this is the proper place to add static template files that are not bundled.